utils/mt5311/files/ebm-read

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+lua /usr/lib/lua/mt5311/ebm-read.lua "$@"
```

Comment:
if you use instead `exec lua /usr/lib/lua/mt5311/ebm-read.lua "$@"` it should consume less RAM...mostly